lib/jumubase_web/templates/performance/new.html.eex

```diff
@@ -35,7 +35,10 @@
       <h3><%= gettext("Participants") %></h3>
 
       <div class="panel-group" id="participant-accordion" role="tablist">
-        <div class="panel panel-default" v-for="(appearance, index) in appearances">
+        <div
+          class="panel"
+          v-for="(appearance, index) in appearances"
+          v-bind:class="getAppearancePanelClass(index)">
```

Comment:
@denisw I'm not sure if this is the way to go here. I saw that Vue has all kinds of class helpers and computed properties, but didn't manage to pass our `index` variable to those…

Reply:
@richeterre I'm pretty sure this is the right approach. My understanding is that `index` is a template-local variable, so passing it to a method is the only working approach.

But alternatively, you could push the panel into the control of the `AppearanceFields` component. Then you could use a computed property because `index` does not need to be passed (it's a prop there, accessible as `this.index`).